docs/health/health_design/design.rst

```diff
@@ -23,7 +80,6 @@ Here is the pyATS Health Check yaml. It's almost same with `Blitz`! There are a
       source:
         pkg: genie.libs.health
         class: health.Health
-      devices: ['uut']
```

Comment:
Is this no longer needed? See my comment above, my understanding was the devices need to in both health.yaml and testbed.yaml

Reply:
ok to have `device` under `action` instead in case of Blitz